pyproject.toml

```diff
@@ -25,6 +25,9 @@ typing-extensions = "^3.7"
 libusb1 = "^1.7"
 pyside2 = { version = "^5.14.0", optional = true }
 bitbox02 = ">=5.3.0,<6.0.0"
+cbor = "^1.0.0"
+requests = "^2.25.1"
```

Comment:
`requests` is a disallowed dependency.

It is my understanding that this implementation will forego the PIN server and instead require the user to enter the mnemonic each time?

Reply:
Will make 'requests' an optional dependency in the api code and will remove from the setup/project files.